fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -223,6 +246,22 @@ static StorageTeamID decodeStorageTeamIDPoppedKey(KeyRef key) {
 	return BinaryReader::fromStringRef<StorageTeamID>(key, Unversioned());
 }
 
+static std::unordered_map<ptxn::TLogGroupID, std::map<StorageTeamID, std::vector<Tag>>> decodePersistStorageTeamValue(
```

Comment:
Where is this function used (and also above `persistStorageTeamValue()`)?

Reply:
no we do not need it, I will remove it, added it at first because I thought about we may need to persist it in the TLog server level, rather than TLog group leve.